js/common/modules/@arangodb/graph/graphs-generation.js

```diff
@@ -253,10 +253,97 @@ const graphGenerator = function (verticesEdgesGenerator) {
                 }
                 break;
         }
-        // console.warn(edges);
         return {vertices, edges};
     };
 
+    /**
+     * Creates a grid with dimensions numberLayers x thickness. If parameter kind is "directed", the edges go from
+     * Layer i to Layer i+1. Edges between vertices of the same layer all go to the same direction and edges of two
+     * layers go to the same direction. If parameter kind is "zigzag", edges within one layer go to the same direction,
+     * but edges between neighbor layers go to the opposite directions. If parameter
+     * kind is "bidirected", the graph is undirected. The vertices are enumerated layer-wise: the first layer gets
```

Comment:
wouldn't it be simpler to either call it `bidirected` or `undirected`, not both?